tailwind.config.js

```diff
@@ -46,6 +46,15 @@ module.exports = {
         'price-default': '#21221F',
         'price-sale': '#EE4266',
       },
+      screens: {
+        '2xsmall': '320px',
+        xsmall: '512px',
+        small: '1024px',
+        medium: '1280px',
+        large: '1440px',
+        xlarge: '1680px',
```

Comment:
This file is to extends tailwind current config. Tailwind has breakpoint `xl`, so your new extended `xlarge` might be confused and not consistent. We are using tailwind recommended breakpoints, so no need to change the whole projects to the new system.

https://tailwindcss.com/docs/responsive-design